README.md

```diff
@@ -15,10 +15,16 @@ File issues to let us know what you'd like to learn. See [Contributing](#contrib
 - [Contribute to Node.js Core](http://nodetodo.org/getting-started/)
 - [Contribute to Node.js (but not sure how to get to the working group you want to help?)](https://github.com/nodejs/getting-started/blob/master/contribute_to_node.md)
 
-
-### Help
-If you're looking for help while writing Node.js, ask questions in the Node.js [Help](https://github.com/nodejs/help#-help) repository. No question is too small!
-
+ ## Support
```

Comment:
-1 on changing "Help" to "Support" since this could lead to confusion for folks who are looking for "commercial support", which this is absolutely not. I suggest leaving this as is with "Help".